pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -140,11 +196,17 @@ public void Validate()
             var time = DateTime.MaxValue;
             var logIndex = int.MinValue;
 
+            //make sure the entries timestamp in the log is always increasing 
             foreach (var entry in AuditLogEntries.Enumerate())
             {
-                Assume.IsTrue(entry.TimeStampUTC <= time && entry.LogIndex > logIndex,
-                    AuditLogStrings.AuditLogList_Validate_Audit_log_is_corrupted__Audit_log_entry_time_stamps_and_indices_should_be_decreasing);
-
+                if (entry.TimeStampUTC > time || entry.LogIndex <= logIndex)
+                {
+                    string msg = string.Format(
+                        AuditLogStrings.AuditLogList_Validate_Audit_log_is_corrupted__Audit_log_entry_time_stamps_and_indices_should_be_decreasing,
+                        entry.LogIndex, entry.TimeStampUTC, logIndex, time);
+                    Exception e = new AuditLogException(msg);
+                    throw e;
```

Comment:
Probably just do:
throw new AuditLogException(msg)
(There are a couple of other places where you split the construction of an exception and the throwing onto separate lines, but I cannot think of a reason that you would want to do that).